src/main/resources/lang/en-US.yml

```diff
@@ -636,4 +581,24 @@ status_nation_peaceful_occupation_tax: "&2Peaceful Occupation Tax: &a%s"
 msg_err_peaceful_occupation_tax_cannot_be_more_than: "Nations cannot set a peaceful occupation tax greater than %s."
 
 #Shown to nation when they are paid their peaceful occupation taxes.
-msg_peaceful_occupation_taxes_paid_totaling: "Your nation received a peaceful occupation taxes totalling: %s."
\ No newline at end of file
+msg_peaceful_occupation_taxes_paid_totaling: "Your nation received a peaceful occupation taxes totalling: %s."
+
+#Added/Updated in 0.51
+#Invasion

```

Comment:
Crowdin will interpret the `# words` above a line as a comment/context that is shown on crowdin.

Having an extra space under `#Invasion` should prevent that, ideally we actually have a comment for each new language string. (Something Towny does most of the time now.)

Reply:
Resolved